controllers/clustermaster_controller.go

```diff
@@ -150,10 +150,10 @@ func (r *ClusterMasterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 // recordInstrumentionData Record api profiling information to prometheus
-func recordInstrumentionData(start time.Time, req ctrl.Request, module string, name string) {
-	metricLabels := getPrometheusLabels(req, name)
-	metricLabels[labelModuleName] = module
-	metricLabels[labelMethodName] = name
-	value := float64(time.Since(start) / time.Millisecond)
-	apiTotalTimeMetricEvents.With(metricLabels).Set(value)
-}
+//func recordInstrumentionData(start time.Time, req ctrl.Request, module string, name string) {
```

Comment:
Are we avoiding the collection of data for already running instances this way?

Reply:
This function is now shared between Master and Manager, because it is generic we only need one declaration. I moved it to `clustermanager_controller.go` so no other action is needed when the Master is deprecated.

Reply:
I will remove from here in the next commit.